shanoir-uploader/src/main/java/org/shanoir/uploader/action/init/SelectProfilePanelActionListener.java

```diff
@@ -31,6 +32,26 @@ public void actionPerformed(ActionEvent e) {
 		String selectedProfile = (String) selectProfilePanel.selectProfileCB.getSelectedItem();
 		ShUpConfig.profileSelected = selectedProfile;
 		configureSelectedProfile(selectedProfile);
+		
+		// If the "Remember profile" box is ticked, we store the selected profile in basic.properties 
+		// to avoid displaying the Profile selection the next time the application starts
+		if (selectProfilePanel.rbRememberProfile.isSelected()) {
+			logger.info("Saving Profile selected in basic.properties file.");
+			String fileName = ShUpConfig.shanoirUploaderFolder + File.separator + ShUpConfig.BASIC_PROPERTIES;
+			final File propertiesFile = new File(fileName);
+			boolean propertiesExists = propertiesFile.exists();
+			if (propertiesExists) {
+				try {
+					Properties props = ShUpConfig.basicProperties;
+					props.setProperty(ShUpConfig.PROFILE, selectedProfile);
+					// Store the new Profile configuration in the basic.properties file
+					OutputStream out = new FileOutputStream(propertiesFile);
```

Comment:
Hi Laurent, good PR, after my code review. I will have to test locally, but it looks very promising.
Please put try (new FileOutputStream) to close the stream correctly? Thank you and with kind regards, Michael

Reply:
I just pushed a commit with an out.close(); at the end of the try.

Reply:
Hi Laurent, thank you! I would propose this: try (OutputStream out = new FileOutputStream(propertiesFile)) {
xxx }, to be sure, as with finally before, that the stream is closed at any moment. Hope it works!

Reply:
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Reply:
Thanks, I just pushed the correction